src/solvespace.cpp

```diff
@@ -466,6 +466,15 @@ void SolveSpaceUI::MenuFile(Command id) {
             SS.GetFilenameAndSave(/*saveAs=*/false);
             break;
 
+		case Command::SAVE_WITH_EXPORTS:
+			SS.GetFilenameAndSave(/*saveAs=*/false);
+
+			SS.ExportAsPngTo(SS.saveFile + ".png");
+			SS.ExportMeshTo(SS.saveFile + ".html");
+			SS.ExportMeshTo(SS.saveFile + ".stl");
```

Comment:
I don't think this is generic enough to be merged. This is just a set of arbitrary three formats that are useful to you, and aren't configurable in any way.